src/types/user.ts

```diff
@@ -516,6 +516,23 @@ export default /* GraphQL */ `
     id: ID!
     address: String!
     createdAt: DateTime!
+
+    """NFT assets owned by this wallet address"""
+    assets: [NFTAsset!]!
+  }
+
+  """ NFT Asset """
+  type NFTAsset {
+    tokenId: String!
+    name: String!
+    description: String
+    imageUrl: String!
+    imagePreviewUrl: String
+    imageThumbnailUrl: String
+    imageOriginalUrl: String!
+    contractAddress: String!
+    collectionName: String!
+    tokenMetadata: String
```

Comment:
We should strong type `tokenMetadata` if it's a key-value JSON string?

e.g.
```
tokenMetadata: [NFTMetaDataItem!]

type NFTMetadataItem {
    name: String!
    value: String
}
```

Reply:
it's a link actually,
```json
            "token_metadata": "https://ipfs.io/ipfs/QmY5cedmZ27MULssFNnVPtYXwA8uYXUY8BiSaBgDkZ98cR/124",
```

may want the traits?
```json
            "traits": [
                {
                    "trait_type": "Ideology",
                    "value": "Epicurean",
                    "display_type": null,
                    "max_value": null,
                    "trait_count": 3,
                    "order": null
                },
                {
                    "trait_type": "Race",
                    "value": "Valya",
                    "display_type": null,
                    "max_value": null,
                    "trait_count": 25,
                    "order": null
                },
                {
                    "trait_type": "Character",
                    "value": "introverted",
                    "display_type": null,
                    "max_value": null,
                    "trait_count": 8,
                    "order": null
                }
            ],

```

Reply:
I think our design doesn't include traits now, so should be fine without it.